whipper/command/cd.py

```diff
@@ -341,7 +350,19 @@ def doCommand(self):
         else:
             logger.info("creating output directory %s", dirname)
             os.makedirs(dirname)
-
+        self.coverArtPath = None
+        if (self.options.fetch_cover_art in {"embed", "complete"} and
+                importlib.util.find_spec("PIL") is None):
+            logger.warning("the cover art option '%s' won't be honored "
+                           "because the 'pillow' module isn't available",
+                           self.options.fetch_cover_art)
```

Comment:
If the option is "complete", it seems like it will still be partially honoured. This message might benefit from being reworded to just stating that the image won’t be embedded. If "embed" is chosen, that’s end-of-road. If "complete" is chosen, that still leaves saving the file.

Reply:
I don't want to have an option value which promises to save a standalone file + embed to appear to work as expected when it does not (only standalone file saved in case of missing `Pillow` dependency). The current code will trigger the warning in that case: the code block below shouldn't be executed because of the `elif`.